src/api/transaction/suspay-finish.js

```diff
@@ -0,0 +1,48 @@
+/* @flow */
+'use strict';
+const _ = require('lodash');
+const utils = require('./utils');
+const validate = utils.common.validate;
+const Transaction = utils.common.core.Transaction;
+
+function createSusPayFinishTransaction(account, payment) {
+  validate.address(account);
+  validate.suspendedPaymentFinish(payment);
+
+  const transaction = new Transaction();
+  transaction.susPayFinish({
+    from: account,
+    owner: payment.owner,
+    offerSequence: payment.offerSequence
```

Comment:
I reused the existing `SField` instead of creating a new one. If there are sound objections, I'm open to changing it.